CHANGELOG.md

```diff
@@ -2,13 +2,181 @@
 
 ## 10.0.0
 
-Teleport 10.0 is a major release of Teleport that contains new features, improvements, and bug fixes.
+Teleport 10 is a major release that brings:
 
-### Breaking Changes
+* Passwordless (Preview)
+* Resource Access Requests (Preview)
+* Proxy Peering (Preview)
+* IP-Based Restrictions (Preview)
+* Automatic User Provisioning (Preview)
+* Audit Logging for Microsoft SQL Server Database Access
+* Snowflake Database Access (Preview)
+* ElastiCache/MemoryDB Database Access (Preview)
+* Teleport Connect for Server and Database Access (Preview)
+* Machine ID Database Access Support (Preview)
```

Comment:
Nit: can we split these by product? IMO, it will add readability.

E.g.

Platform Features:

- Passwordless Registration and Login (Preview)
- Resource Access Requests (Preview)
- Proxy Peering (Preview)

Server Access Features:
- IP-Baesd Restrictions (Preview)
- Automatic User Provisioning (Preview)

etc.